packages/mobile/src/screens/sign-up-screen/utils/socialMediaMessages.ts

```diff
@@ -0,0 +1,15 @@
+export const socialMediaMessages = {
```

Comment:
Maybe I should move these to common actually 🤔

Reply:
would love that. something like common/src/messages/sign-up/socialMediaMessages.ts?